build/linux/installer/scripts/livenessprobe.sh

```diff
@@ -1,4 +1,28 @@
-#!/bin/bash
+#!/bin/bash 
```

Comment:
Hi Janvi, Can you please post the before and after resource utilizations on the same cluster, before & after this change ?

Reply:
Added the images in description. Let me know if you need me to add more information